webapi/src/main/twirl/org/knora/webapi/messages/twirl/queries/sparql/admin/setProjectRestrictedViewSettings.scala.txt

```diff
@@ -0,0 +1,49 @@
+@*
+ * Copyright © 2021 - 2023 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ *@
+
+@import org.knora.webapi.IRI
+@import dsp.valueobjects.RestrictedViewSize
+
+@*
+ * Sets project restricted view settings.
+ *
+ * @param iri project's IRI.
+ * @param size project's restricted view size.
+ * @param watermark project's watermark.
+ *
+ *@
+@(iri: IRI,
+  size: String,
+  watermark: Option[String])
```

Comment:
The `watermark` argument is always passed in as `None`.
Do we really need this parameter? Seem to me like it is unused and untested code.

Reply:
Good question. I made it the way to have one template for both fields of `RestrictedViewSettings`, however watermark is not yet supported. So now either:
1. I leave it as it is.
2. Remove unused field in the template only.
3. Above + rename entire route, methods and classes from  set `RestrictedViewSettings` to `RestrictedViewSize`, because it is only about the size now.

What do you think? 1, 2 or 3?

Reply:
I would prefer the second option. We can always add the `watermark` support to the route later if we need to.